skins/cat17/templates/Base_Layout.html.twig

```diff
@@ -37,12 +37,14 @@
 		{% include 'partials/' ~ footer_template %}
 	</div>
 
+	{% block scripts %}
```

Comment:
I think it would be beneficial to have a _separate_ script block, independent of the scripts we always have to load (which should be as small as possible but something tells me we'll always have it), in order not to inadvertently overwrite them (forgotten `{$ parent() $}`). What do you think?